angr/analyses/decompiler/region_simplifiers/expr_folding.py

```diff
@@ -163,7 +163,7 @@ class ExpressionUseFinder(AILBlockWalker):
 
         v16 = ((int)v23->field_5) + 1 & 255;
         v23->field_5 = ((char)(((int)v23->field_5) + 1 & 255));
-        v13 = printf("Recieved packet %d for connection with %d\\n", v16, a0 & 255);
+        v13 = printf("Received packet %d for connection with %d\\n", v16, a0 & 255);
```

Comment:
It was misspelled as "Recieved" in the original binary.

Reply:
undone.